core/src/providers/anthropic.rs

```diff
@@ -1185,25 +1269,6 @@ impl LLM for AnthropicLLM {
             },
         );
 
-        messages = messages
```

Comment:
Hey @spolu, do you recall the reason behind always using a single item in the content Vec? This was introduced in https://github.com/dust-tt/dust/pull/4180.

Reply:
No good reason IMHO.

Did you double check the code below (we need to keep it around for now due to old assistant conversation rendering) but just want to make sure it won't interfere too much ?

Reply:
Sorry looks like it's the code above :)

Since you updated the comment. Looks good

Reply:
Yep from my tests no regression are introduced by removing this logic.